aws_connect.py

```diff
@@ -1,28 +1,173 @@
+import os
+import time
 import boto3
-import configparser
 import botocore.exceptions
-from fastapi import FastAPI, HTTPException
+import psycopg2
+from fastapi import FastAPI, HTTPException, Depends, status
+from typing import Annotated
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from pydantic import BaseModel
+from datetime import datetime, timedelta, timezone
+from jose import JWTError, jwt
+from passlib.context import CryptContext
 
+SECRET_KEY = os.environ["SECRET_KEY"]
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
 
+for db_connect_attempts in range(10):
+    try:
+        db_connect = psycopg2.connect(
+            host=os.environ["DB_IP"],
+            database=os.environ["DB_NAME"],
+            user=os.environ["DB_USER"],
+            password=os.environ["DB_PWD"],
+        )
+    except psycopg2.OperationalError as error:
+        print(error)
+        time.sleep(1)
+    else:
+        break
+with db_connect.cursor() as cursor:
+    cursor.execute("SELECT * from users")
+    row = cursor.fetchone()
+    while row is not None:
+        user_db = {
+            row[0]: {"username": row[0], "hashed_password": row[1], "disabled": row[2]}
+        }
+        row = cursor.fetchone()
```

Comment:
we do not need to keep the user database in program memory, we can look up user only when needed (in `get_user`)